### ISSUE TITLE
opt_user unused in manhole twistd plugin

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5283]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)| @teratorn reported|
|-|-|
|Trac ID|trac#5283|
|Type|defect|
|Created|2011-09-29 05:07:40Z|
|Branch|https://github.com/twisted/twisted/tree/remove-opt-user-5283|

Some quick digging and grepping indicates that twisted.conch.manhole_tap.Options.opt_user and the .users list is dead code.

<details><summary>Searchable metadata</summary>

```
trac-id__5283 5283
type__defect defect
reporter__teratorn teratorn
priority__normal normal
milestone__ 
branch__branches_remove_opt_user_5283 branches/remove-opt-user-5283
branch_author__teratorn teratorn
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1317272860000000 1317272860000000
changetime__1317455109000000 1317455109000000
version__None None
owner__teratorn teratorn

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_1"></a>|@teratorn commented|
|-|-|

(In [32686]) Branching to 'remove-opt-user-5283'

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_2"></a>|@teratorn commented|
|-|-|

All tests still pass for me. Ready for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @teratorn|
|-|-|

If we slavishly adhere to the compatibility policy, this should get a deprecation first.  However, there's no functionality; no one could ever have benefited from using this option.  If anyone was using it, it is trivial for them to fix their code - stop calling the method or stop passing the argument.

So I'm okay with skipping the deprecation.  Please do add a removal [ReviewProcess#Newsfiles news fragment] though.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_5"></a>|@teratorn commented|
|-|-|

(In [32714]) add news file
Refs: [#5283](https://github.com/twisted/twisted/issues/5283)

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_6"></a>|@teratorn removed owner|
|-|-|

OK

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph commented|
|-|-|

[build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-opt-user-5283)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_8"></a>|@glyph set owner to @teratorn|
|-|-|

Looks good to me, thanks for noticing this.  I echo exarkun's sentiment about this being acceptable to fast-track for removal - if anyone depends on this hopefully they'll notice in a prerelease.

Go ahead and land.

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_9"></a>|@teratorn set status to `closed`|
|-|-|

(In [32723]) Merge remove-opt-user-5283: --user option to `twistd manhole' was dead code

Author: teratorn
Reviewer: exarkun, glyph
Fixes: [#5283](https://github.com/twisted/twisted/issues/5283)
